apps/i18n/common/en_us.json

```diff
@@ -60,7 +60,6 @@
   "censusFrequency1to3": "1-3 hours per week",
   "censusFrequency3plus": "3+ hours per week",
   "censusHeading": "Tell us about your school",
-  "censusHowMuch": "How much coding/computer programming is taught at this school? (assume for the purposes of this question that this does not include HTML/CSS, Web design, or how to use apps)",
```

Comment:
We could leave this in here if we want it loc'ed for later, but otherwise no problem.